packages/client/src/constants/game.ts

```diff
@@ -0,0 +1,84 @@
+const canvas = {
+  width: 800,
+  height: 360,
+}
+export {canvas}
+
+const SpriteCat = {
+  width: 100,
+  height: 74,
+  aspectRatio: 0.74,
+}
+export {SpriteCat}
+
+// Base game constants
+const scorePerLevel = 100
+const initialScore = 10	// Need to prevent 'Game over' after the first fail
+const ActionPositionVertical = Math.floor(canvas.height * 0.93)
+const shadowsEnable = false
+const trajectoryStep = 2
+const jumpHeightMin = Math.floor(SpriteCat.height / 1.5)
+const jumpHeightMax = SpriteCat.height * 3
+const defaultCatX = Math.floor(canvas.width / 3)
+const defaultCatY = ActionPositionVertical
+const defaultTargetX = canvas.width / 2
+const victimPositionDelta = canvas.width / 8
+const defaultTargetY = ActionPositionVertical
+const defaultTargetHeight = 80
+const stepTargetHeight = 10
+const defaultTargetDelay = 8000 // The time after which the target will escape
+const stepTargetDelay = 1000
+
+export type TargetName = 'mouse' | 'grasshopper' | 'butterfly' | 'bird' | 'cactus' | 'puddle' | 'flowerpot' | 'gnome' | 'none'
+
+const VICTIM_LIST: TargetName[] = ['mouse', 'grasshopper', 'butterfly', 'bird']
+export {VICTIM_LIST}
```

Comment:
не стоит export-ы отдельно прописать)) 
```suggestion
export const VICTIM_LIST: TargetName[] = ['mouse', 'grasshopper', 'butterfly', 'bird']
```